.babelrc

```diff
@@ -0,0 +1,3 @@
+{
+  "presets": ["react", "es2015", "stage-0"]
```

Comment:
Also we should be using 
```
"presets": [
  ["es2015", { "modules": false }], "react"
]
```
since webpack handles modules on it's own.  I need to do this on LD as well...